### PR TITLE
po: Added Estonian translation

### PR DESCRIPTION
**Pull request by tenseoverflow**
Added Estonian (et) translation to the project.

### CODE REVIEW

File: po/et.po [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+msgstr "Laiendushaldur (Arendajale)"
+
+#: src/exm-application.c:107
+msgid "Extension Manager"
+msgstr "Laiendushaldur"
+
+#. TRANSLATORS: 'Name <email@domain.com>' or 'Name https://website.example'
+#: src/exm-application.c:117
+msgid "translator-credits"
+msgstr "Henri <contact@hen.ee>, 2022"
```
Comment (mjakeman) [inline po/et.po:143]:
```suggestion
msgstr "Henri <contact@hen.ee>"
```

This shouldn't have a year here (as it breaks the email formatting)

File: po/et.po [hunk 2]
```diff
@@ -0,0 +1,351 @@
...
+msgstr "Kasutaja arvustused"
+
+#: src/exm-detail-view.blp:187
+msgid "Show All Reviews"
+msgstr "Näita kõiki arvustusi"
+
+#. Translators: Use unicode ellipsis '…' rather than three dots '...'
+#: src/exm-detail-view.c:364
+msgid "Loading…"
+msgstr "Laadimine..."
```
Comment (mjakeman) [inline po/et.po:219]:
```suggestion
msgstr "Laadimine…"
```

Please use the ellipsis character '…' instead of three dots.

**Review by mjakeman — CHANGES_REQUESTED** [review]
Hi, thanks for the contribution!

There are a few minor issues but otherwise looks good to me.

**Comment by tenseoverflow** [comment]
Thanks for your review! I've gone ahead and committed your changes.

**Comment by mjakeman** [comment]
Great, thank you!

**Review by mjakeman — APPROVED** [review]

**Pull request merged**